pytorch_blade/pytorch_blade/torch-mlir/lib/Dialect/TorchConversion/Transforms/DiscDecomposeComplexOps.cpp

```diff
@@ -134,6 +134,50 @@ LogicalResult ConvertAtenOp<OperatorOp>::matchAndRewrite(
     rewriter.replaceOpWithNewOp<AtenDivTensorOp>(
         op, outTy, op.getOperand(0), op.getOperand(1));
     return success();
+  } else if ("aten.max.other" == name) {
+    auto outTy = op.getResult(0).getType();
+    rewriter.replaceOpWithNewOp<AtenMaximumOp>(
+        op, outTy, op.getOperand(0), op.getOperand(1));
+    return success();
+  } else if ("aten.narrow.Tensor" == name) {
+    // start must be an 0-dim integral Tensor.
+    auto outTy = op.getResult(0).getType();
+    auto start = rewriter.create<AtenItemOp>(
+        loc, Torch::IntType::get(op.getContext()), op.getOperand(2));
+    rewriter.replaceOpWithNewOp<AtenNarrowOp>(
+        op, outTy, op.getOperand(0), op.getOperand(1), start, op.getOperand(3));
+    return success();
+  } else if ("aten.selu" == name || "aten.selu_" == name) {
```

Comment:
just curious these ops are not defined in torch-diaclect?

Reply:
`torch-dialect` hasn't defined all the operators from the torch, it needs time.